### DISCUSSION TITLE
Pig Fu

### DISCUSSION THREAD

**Discussion by diegostoliar** [keiwando/toonsquid#500] [Show and tell]
I,m having some fun time doing those small animations to my website

[pig fu kick](https://www.stoliar.art/watch/anim002)

**Comment by Cat-Butler** [comment]
Is that what they mean by a ham actor? (Sorry.)
That animation was fast as lightning (ahem), but you could still make out the action well - nice timing. 
Kudos!

**Reply by diegostoliar** [reply]
Thanks :)
